packages/core/styles/CHANGELOG.md

```
@@ -2,6 +2,16 @@
 
 Nimbus Styles deprive all styles needed to build nimbus components.
 
+## 2026-08-27 `9.68.1`
+
+#### 🛠 Breaking changes
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)